packages/lab/src/__tests__/__e2e__/stepper-input/StepperInput.cy.tsx

```diff
@@ -3,33 +3,13 @@ import { StepperInput } from "@salt-ds/lab";
 describe("Stepper Input", () => {
```

Comment:
- Need to add tests around keyboard interactions, e.g. arrow up / down?
- What about states, e.g. disable?
- Controlled use cases?
- `sanitizedInput`?